From 50ef0f15725b27c404b95a4eae5d15a96a5d847f Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:03:23 -0400
Subject: [usb] wusbcore: fix up line break coding style issues in mmc.c

Message-id: <1410887463-20674-132-git-send-email-dzickus@redhat.com>
Patchwork-id: 95278
O-Subject: [RHEL7 PATCH 131/591] USB: wusbcore: fix up line break coding style issues in mmc.c
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 486513179931649977edf24dfbde7d82e3d904b2
Author: Rahul Bedarkar <rahulbedarkar89@gmail.com>
Date:   Sat Jan 4 14:06:31 2014 +0530

    USB: wusbcore: fix up line break coding style issues in mmc.c

    Signed-off-by: Rahul Bedarkar <rahulbedarkar89@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/wusbcore/mmc.c b/drivers/usb/wusbcore/mmc.c
index b71760c..4474126 100644
--- a/drivers/usb/wusbcore/mmc.c
+++ b/drivers/usb/wusbcore/mmc.c
@@ -206,13 +206,15 @@ int wusbhc_start(struct wusbhc *wusbhc)
 
  result = wusbhc_devconnect_start(wusbhc);
  if (result < 0) {
-  dev_err(dev, "error enabling device connections: %d\n", result);
+  dev_err(dev, "error enabling device connections: %d\n",
+   result);
   goto error_devconnect_start;
  }
 
  result = wusbhc_sec_start(wusbhc);
  if (result < 0) {
-  dev_err(dev, "error starting security in the HC: %d\n", result);
+  dev_err(dev, "error starting security in the HC: %d\n",
+   result);
   goto error_sec_start;
  }
 
@@ -284,7 +286,8 @@ int wusbhc_chid_set(struct wusbhc *wusbhc, const struct wusb_ckhdid *chid)
   wusbhc->uwb_rc = uwb_rc_get_by_grandpa(wusbhc->dev->parent);
   if (wusbhc->uwb_rc == NULL) {
    result = -ENODEV;
-   dev_err(wusbhc->dev, "Cannot get associated UWB Host Controller\n");
+   dev_err(wusbhc->dev,
+    "Cannot get associated UWB Host Controller\n");
    goto error_rc_get;
   }
 
-- 
1.7.1